src/docs/Foundations/Layout/Grid/index.mdx

```diff
@@ -7,11 +7,11 @@ status:
   scss: 'stable'
 ---
 
-> A grid is used to create a **more structured and more harmonious design.** It’s the frame of your page, you build the layout by placing blocks of contents (text, images,…) on a certain number of columns. Gutters are used to separate 2 blocks, they give air to the design. Columns and gutters form a container. The container can be **full width or with a fixed size.**
+> Mozaic Flex grid system is a fully responsive, customizable, 12 column layout system with five default breakpoints, Sass variables and mixins, and dozens of predefined classes.
```

Comment:
In my opinion, we should keep the old definition, which is a good detail of how a grid works. We should just rephrase perhaps some of the sentences.